src/app/plugins/wolframalpha.plugin.ts

```diff
@@ -0,0 +1,73 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import Environment from '../../environment';
+// @ts-ignore
+import WolframAlphaAPI from 'wolfram-alpha-api';
+
+export class WolframAlphaPlugin extends Plugin {
+  private _defaultQuestion = 'What can you do?';
+  private _imageOptions = ['image', 'img'];
+  private _logoURL = 'https://www.symbols.com/images/symbol/2886_wolfram-alpha-logo.png';
+  private _errorMessage = "Sorry, I don't know that one";
+
+  public name: string = 'Wolfram Alpha';
+  public description: string =
+    'Ask wolfram alpha a question. \nProvide the first argument to get your answer as an image';
+  public usage: string = `wa <${this._imageOptions.join(' | ')}>? <question>`;
+  public pluginAlias = ['wa', 'wolfram', 'alpha', 'wolframalpha'];
+  public permission: ChannelType = ChannelType.Public;
+
+  constructor(public container: IContainer) {
+    super();
+    this.container.waApi = WolframAlphaAPI(Environment.WolframAppID);
```

Comment:
I think the idea was to assign this to a field in the class, rather the container, since the container is meant for all plugins.

Reply:
@joey-colon

Reply:
yeah, just `private _waAPI;`
then in constructor `this.waAPI = ...`

Reply:
+1 to what @suneettipirneni  and @tanndlin said